ns/time.n3

```diff
@@ -0,0 +1,173 @@
+#   n3 definition of some time and date functions
+#
+#  Based on orginal cwm_date and cwm_time by Mark Nottingham
+# http://www.mnot.com/python
+# These use isodate.py and timegm.py, by mnot.
+# Released with permission under W3C code licence.
+#
+# Schema (c) TimBL, #C code licence.
+# 
+# $Id: time.n3,v 1.7 2006-12-13 21:10:29 timbl Exp $
+
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix doc: <http://www.w3.org/2000/10/swap/pim/doc#>.
+@prefix log: <http://www.w3.org/2000/10/swap/log#>.
+@prefix math: <http://www.w3.org/2000/10/swap/math#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix vs: <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
+
+@prefix time: <http://www.w3.org/2000/10/swap/time#>.
+@prefix : <#>.
+
+<> <http://www.w3.org/2000/10/swap/pim/doc.n3#licence> <http://www.w3.org/Consortium/Legal/copyright-software-19980720>.
+
+
+time:inSeconds a rdf:Property, owl:FunctionalProperty, owl:ReverseFunctionalProperty;
```

Comment:
shouldn't this be owl:InverseFunctionalProperty? ..

Reply:
```suggestion
time:inSeconds a rdf:Property, owl:FunctionalProperty, owl:InverseFunctionalProperty;
```